_includes/landing-navbar.html

```diff
@@ -21,8 +21,11 @@
             class="navbar-brand navbar-brand-beta"
             style="padding-left: 0"
             href="{{ landing_page_base_url }}"
-            ><strong>Gruntwork</strong>&nbsp;Labs</a
-          >
+            >
+            {% if page.extra_title %} <strong> {{ page.extra_title }} </strong> &nbsp;|&nbsp; Gruntwork&nbsp;Labs
+            {% else %} <strong>Gruntwork</strong>&nbsp;Labs
+            {% endif %}
```

Comment:
Here's a slightly tweaked layout for the text to consider:

<img width="563" alt="Screen Shot 2021-08-18 at 9 42 36 AM" src="https://user-images.githubusercontent.com/711908/129867233-e55e6a67-dd47-4192-9ba9-5fc9a17e6122.png">

This way, we get more focus on the name of the product, Patcher, as it's now `font-size: 50px`, and stands out more from the rest.

Reply:
Deal! It does differ from all the other landing pages, but since it's not a service, but more a standalone product, I think it's fine.